feature-libs/order/assets/translations/en/my-account-v2-order.i18n.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const myAccountV2Order = {
+  myAccountV2OrderHistory: {
+    heading: 'All Orders ({{param}})',
+    item: '{{param}} Item',
+    items: '{{param}} Items',
+    totalPrice: 'Total Price: {{param}}',
+    consignmentCode: 'Consignment {{param}}',
+    statusDate: 'Last Updated: {{param}}',
+    returnProcessed: 'Return Processed: {{param}}',
+    deliveryPointOfServiceDetails: {
+      itemsToBePickUp: 'To Be Picked Up - ',
+    },
+    checkoutMode: {
+      deliveryEntries: 'To Be Shipped - ',
+    },
+    checkoutPickupInStore: {
+      heading: 'To Be Picked Up - ',
+    },
+    orderListResults: 'Orders List',
+    orderListPagination: 'Orders List pagination',
+    totalPriceLabel: 'Total Price',
+    orderPlaced: 'Order Placed On',
+    orderCodeLabel: 'Order Code',
+    consignmentDetailLabel: 'Consignment Information',
+    consignmentNumberLabel: 'Consignment Number',
+    consignmentStatusLabel: 'Consignment Status',
+    consignmentStatusDateLabel: 'Last Updated On',
+    estimateDeliveryLabel: 'Estimated Delivery Date',
+  },
+  myAccountV2OrderDetails: {
+    returnItems: 'Return Items',
+    cancelItems: 'Cancel Items',
+    downloadInvoices: 'Download Invoices',
+    viewAllOrders: 'View All Orders',
+    noInvoices: 'Invoices are not generated yet. Please come back later.',
+  },
+  myAccountViewOrder: {
```

Comment:
[QUESTION]
Only now I noticed that this translation key namespace doesn't have `myAccountV2` in it's prefix.

Other namespaces have it:
`myAccountV2ConsignmentTracking`
`myAccountV2OrderHistory`
`myAccountV2OrderDetails`

What do you think about having here `myAccountV2ViewOrder` for consistency? So other teams taking this PR as an example will have no confusion about "when to add the prefix and when not". The answer would be consistent: for everything that is related to the new my account.

Reply:
done, changed the name to myAccountV2Orders